<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Navigate</class>
 <widget class="QDialog" name="Navigate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="back_pushButton">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>560</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="algo_groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>121</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Algorithm</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QComboBox" name="algorithm_comboBox"/>
    </item>
    <item>
     <widget class="QPushButton" name="run_pushButton">
      <property name="text">
       <string>Run</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Back_choose_pushButton">
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="Destination_groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>161</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>Destination</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>from</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="from_comboBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>to</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="to_comboBox"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="Execute_pushButton">
      <property name="text">
       <string>Execute</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
